src/test/java/seedu/address/logic/parser/FindByTagCommandParserTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindByTagCommand;
+import seedu.address.model.item.TagMatchesKeywordsPredicate;
+
+public class FindByTagCommandParserTest {
+    private FindByTagCommandParser parser = new FindByTagCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                FindByTagCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsFindCommand() {
+        // no leading and trailing whitespaces
+        FindByTagCommand expectedFindCommand =
+                new FindByTagCommand(new TagMatchesKeywordsPredicate(Arrays.asList("Apple", "Banana")));
+        assertParseSuccess(parser, "Apple, Banana", expectedFindCommand);
+
+        // space between same input
+        assertParseSuccess(parser, "Apple , Banana", expectedFindCommand);
```

Comment:
add more spaces

Reply:
what? nah i havent figured out a systematic regex that can do:
"UnlimitedWhiteSpace,UnlimitedWhitespace"
The current pattern used is `" , |, |,"`
If you know how to do dat, i can probably also make the 
findt and find command use the same syntax rather than spaces which atm is q unintuitive in the find cmd